### PR TITLE
Macos compat

### LINKED ISSUE #1: compiling with tic segfaults on macos
macOS doesn't appear to like it when you attempt to install a ti with tic that uses a terminfo it doesn't know about. I don't have an `eterm-color` installed (should I?). If I change `use=eterm-color` to `use=xterm-color` it works fine.

Thanks for this package btw, I've wanted something like this for a while! Now what would it take to get 24-bit support... 😄

### PR DESCRIPTION
**Pull request by aaronjensen**
Fixes #1

### CODE REVIEW

File: eterm-256color.el [hunk 1]
```diff
@@ -40,6 +40,13 @@
   "256colors for term."
   :group 'term)
 
+(defcustom eterm-256color-terminfo-path (pcase system-type
```
Comment (dieggsy) [inline eterm-256color.el:43]: Could you add `:group 'eterm-256color` and `:type 'string` to this?

Comment (aaronjensen) [reply eterm-256color.el:43]: Done.

**Comment by dieggsy** [comment]
So I added the commit to the readme - regarding the file check, I think I have a more robust solution. According to [the manpage for tic](https://linux.die.net/man/1/tic) (I also checked the manpage for osx and openbsd), it either looks in the dir specified by environment variable `TERMINFO` or `~/.terminfo`. Because of this behavior, it may not make sense to make this a defcustom - we have all the options of where the file could be available to us.

It's possible (with f.el for example) to recursively check either TERMINFO if that is set or `~/.terminfo` for the file `eterm-256color`, making the check more general and (ideally) system agnostic. Working on that now.

**Comment by dieggsy** [comment]
Oh, and thanks a lot for testing my package and opening this PR!

**Comment by aaronjensen** [comment]
That sounds like a much better solution. Thanks!

**Pull request closed without merging**